ccpp/data/GFS_typedefs.meta

```diff
@@ -3381,6 +3381,12 @@
   dimensions = ()
   type = real
   kind = kind_phys
+[num_dfi_radar]
+  standard_name = num_dfi_radar
```

Comment:
Some of these new standard names can probably be improved and/or made to be consistent with https://github.com/ESCOMP/CCPPStandardNames/blob/main/StandardNamesRules.rst. For example, what does DFI stand for?

Reply:
The names match those in WRF. I can make them more elaborate in the meta files if that is necessary.

Reply:
The variables now have more satisfying names.